kogito-test-utils/pom.xml

```
@@ -74,6 +74,18 @@
       <version>${version.org.testcontainers}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>oracle-xe</artifactId>
+      <version>${version.org.testcontainers}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc11</artifactId>
+      <version>21.3.0.0</version>
```

Please move this value to a `version.com.oracle.database.jdbc` property in [this file](https://github.com/kiegroup/kogito-runtimes/blob/main/kogito-build/kogito-dependencies-bom/pom.xml#L71) (NOTE: properties are alphabetically ordered)
```suggestion
      <version>${version.com.oracle.database.jdbc}</version>
```